README.rst

```diff
@@ -2523,23 +2524,25 @@ List device monitoring information
 
     GET /api/v1/monitoring/device/
 
-**Available filters**
-
-You can use the ``monitoring_health_status`` filter (e.g. critical, ok, problem, and unknown)
-to obtain a list of device metrics and their corresponding monitoring status.
-
-.. code-block:: text
-
-   GET /api/v1/monitoring/device/?monitoring__status={monitoring_health_status}
-
 **Notes**:
 
 - The response contains device information and monitoring status (health status),
-  but it does not include information about the list of device metrics and their
-  respective statuses. This is done to avoid generating extra queries for each device.
+  but it does not include the information and
+  health status of the specific metrics, this information
+  can be retrieved in the detail endpoint of each device.
 
 - This endpoint can be accessed with session authentication and token authentication.
 
+**Available filters**
+
+Data can be filtered by health status (e.g. critical, ok, problem, and unknown)
+to obtain the list of devices in the corresponding status, for example,
+to retrieve the list of devices which are in critical conditions
+(eg: unreachable), the following will work:
+
+.. code-block:: text
+
+   GET /api/v1/monitoring/device/?monitoring__status=ok
```

Comment:
@nemesisdesign for critical (unreachable) condition, I think it should be : `?monitoring__status=critical`

Reply:
ups sorry, I was too in a hurry, thanks!